diff --git a/node_modules/mongodb-memory-server-core/lib/util/MongoBinaryDownloadUrl.js b/node_modules/mongodb-memory-server-core/lib/util/MongoBinaryDownloadUrl.js
index 5c6293a..e8b37a5 100644
--- a/node_modules/mongodb-memory-server-core/lib/util/MongoBinaryDownloadUrl.js
+++ b/node_modules/mongodb-memory-server-core/lib/util/MongoBinaryDownloadUrl.js
@@ -432,6 +432,9 @@ class MongoBinaryDownloadUrl {
         if (ubuntuYear >= 21 && semver.satisfies(coercedVersion, '<6.0.4')) {
             return 'ubuntu2004';
         }
+        if (ubuntuYear === 23) {
+            return 'ubuntu2204';
+        }
         // TODO: change or remove "14" default, since it no-longer is supported above 4.0
         // the "04" version always exists for ubuntu, use that as default
         return `ubuntu${ubuntuYear || 14}04`;
